### PR TITLE
Update 200423sexshufflev1.py

### PR DESCRIPTION
**Pull request by tkrzewina**
Introduce an orgasm future

### CODE REVIEW

File: 200423sexshufflev1.py [hunk 1]
```diff
@@ -4,9 +4,11 @@
 random.shuffle(sex_acts)
 scene1=sex_acts[0:4]
 print("Scene 1")
-for sex_acts in scene1:
-    print(sex_acts)
+for sex_acts1 in scene1:
+    print(sex_acts1)
```
Comment (lookasu) [inline 200423sexshufflev1.py:8]:
```suggestion
for act in scene1:
    print(act)
```

File: 200423sexshufflev1.py [hunk 2]
```diff
@@ -4,9 +4,11 @@
...
+print("Orgasm happend during " + random.choice(scene1))
 print("Scene 2")
 scene2=sex_acts[4:8]
-for sex_acts in scene2:
-    print(sex_acts)
+for sex_acts2 in scene2:
+    print(sex_acts2)
```
Comment (lookasu) [inline 200423sexshufflev1.py:13]:
```suggestion
for act in scene2:
    print(act)
```

File: 200423sexshufflev1.py [hunk 3]
```diff
@@ -4,9 +4,11 @@
 random.shuffle(sex_acts)
 scene1=sex_acts[0:4]
 print("Scene 1")
-for sex_acts in scene1:
-    print(sex_acts)
+for act in scene1:
+    print(act)
+print("Orgasm happend during " + random.choice(scene1))
```
Comment (lookasu) [inline 200423sexshufflev1.py:9]: What is the purpose of this?

Comment (tkrzewina) [reply 200423sexshufflev1.py:9]: random selection of orgasmic act in given scene

**Pull request merged**